Tests/BasicsTests/SandboxTests.swift

```diff
@@ -1,60 +1,73 @@
 /*
  This source file is part of the Swift.org open source project
 
- Copyright (c) 2021-2022 Apple Inc. and the Swift project authors
+ Copyright (c) 2021 - 2022 Apple Inc. and the Swift project authors
  Licensed under Apache License v2.0 with Runtime Library Exception
 
  See http://swift.org/LICENSE.txt for license information
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
  */
 
-@testable import Basics
+import Basics
 import SPMTestSupport
 import TSCBasic
 import TSCUtility
 import XCTest
 
 final class SandboxTest: XCTestCase {
+
+    func testDefaults() throws {
+        let sandboxProfile = SandboxProfile()
+        XCTAssertEqual(sandboxProfile.allowNetwork, false)
+        XCTAssertEqual(sandboxProfile.allowWritingToTemporaryDirectories, false)
+        XCTAssertEqual(sandboxProfile.pathRules, [])
+    }
+
     func testSandboxOnAllPlatforms() throws {
         try withTemporaryDirectory { path in
-            let command = Sandbox.apply(command: ["echo", "0"], strictness: .default, writableDirectories: [])
+            let command = SandboxProfile().apply(to:
+                ["echo", "0"])
             XCTAssertNoThrow(try Process.checkNonZeroExit(arguments: command))
         }
     }
 
     func testNetworkNotAllowed() throws {
         #if !os(macOS)
-        try XCTSkipIf(true, "test is only supported on macOS")
+        try XCTSkipIf(true, "sandboxing is only supported on macOS")
         #endif
 
-        let command = Sandbox.apply(command: ["ping", "-t", "1", "localhost"], strictness: .default, writableDirectories: [])
```

Comment:
This is blocked anyway if the firewall is on and "stealth mode" is enabled, so this isn't an adequate test of the sandbox.